packages/react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -130,6 +133,29 @@ const Tooltip: TooltipComponent = React.forwardRef(
     const prefix = usePrefix();
     const child = React.Children.only(children);
 
+    const {
+      'aria-label': ariaLabel,
+      'aria-labelledby': ariaLabelledBy,
+      'aria-describedby': ariaDescribedBy,
+    } = child?.props ?? {};
+
+    const hasLabel = !!label;
+    const hasAriaLabel = typeof ariaLabel !== 'undefined' && ariaLabel !== null;
```

Comment:
```suggestion
  
const hasAriaLabel = typeof ariaLabel === 'string' ? ariaLabel.trim() !== '' : false;
```
This would prevent empty or whitespace-only aria-labels from being treated as valid but would be ignored by screen readers

Reply:
Thanks for check that!